templates/patterns/button/pattern-button.html.twig

```diff
@@ -1,18 +1,34 @@
-{% set baseClass = outline ? 'btn-outline-' : 'btn-' %}
-{% set attributes = attributes.addClass([
-  'btn',
-  style ? baseClass ~ style,
-  size ? 'btn-' ~ size,
-  block ? 'btn-block',
-  disabled ? 'disabled'
-  ]) %}
-
-{% if url or attributes.href %}
-  {% set url = url ?: attributes.href %} 
-  <a{{ attributes.setAttribute('href', url) }}>{{ label }}</a>
-{% else %}
-  {% if disabled %}
-    {% set attributes = attributes.setAttribute('disabled', true) %}
-  {% endif %}
-  <button type="button"{{ attributes }}>{{ label }}</button>
+{#
+/**
+ * @file
+ * Button component.
+ */
+#}
+{%  if _icon %}
+  {% set _icon = icon|merge({
+    path: bcl_icon_path
+})  %}
 {% endif %}
+{%  if show_spinner %}
+  {% set spinner = {
+    'type': "border",
+    'variant': "info",
+    'small': true,
```

Comment:
the icon is to close to the text, please add ``attributes: create_attribute().addClass('me-1'),`` like in netlify data tab